sys/auto_init/saul/auto_init_pca9685.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2017 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/*
+ * @ingroup     drivers_pca9685
+ * @ingroup     sys_auto_init_saul
+ * @brief       Auto initialization of PCA9685 I2C PWM controller
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ */
+
+#if MODULE_PCA9685 && MODULE_SAUL
+
+#include "assert.h"
+#include "log.h"
+
+#include "saul_reg.h"
+#include "saul/periph.h"
+
+#include "pca9685.h"
+#include "pca9685_params.h"
+
+/**
+ * @brief   Number of configured PCA9685 DAC devices
+ */
+#define PCA9685_NUM    (sizeof(pca9685_params) / sizeof(pca9685_params[0]))
```

Comment:
There is an `ARRAY_SIZE()` macro now.

Reply:
I know, at that time when I wrote this driver (last year December) there was no macro. I can replace it of course if you insist on it.

Reply:
Just one small fixup ;)

You can squash already.

Reply:
Replaced them by `ARRAY_SIZE` macro.